src/forms/propertiesdialog.ui

```diff
@@ -84,104 +84,7 @@
      </property>
      <widget class="QWidget" name="appearancePage">
       <layout class="QGridLayout" name="gridLayout_3">
-       <item row="8" column="0">
```

Comment:
Could you keep unrelated controls untouched? I know Qt Designer often messed up UI files, so you may need to do that manually.